src/components/courselayout.tsx

```diff
@@ -0,0 +1,14 @@
+const CourseLayout = () => {
+  return (
+    <div className="bg-transparent-80 p-auto font-Malayalam mx-auto flex w-[95vw] flex-col items-center rounded-[1vw] bg-devstation-green-200 p-[2.5vw] text-center font-normal">
```

Comment:
```suggestion
    <div className="bg-transparent-80 w-11/12 flex flex-col items-center rounded-xl bg-devstation-green-200 p-[2.5vw]">
```
a lot of the classes you added are not needed

Reply:
if you're curious as to why, feel free to ask us in the general chat